Python/pywarpx/picmi.py

```diff
@@ -387,9 +387,9 @@ def initialize_inputs(self):
             if self.moving_window_velocity[0] != 0.:
                 pywarpx.warpx.moving_window_dir = 'x'
                 pywarpx.warpx.moving_window_v = self.moving_window_velocity[0]/constants.c  # in units of the speed of light
-            if self.moving_window_velocity[1] != 0.:
-                pywarpx.warpx.moving_window_dir = 'y'
-                pywarpx.warpx.moving_window_v = self.moving_window_velocity[1]/constants.c  # in units of the speed of light
+            if self.moving_window_velocity[2] != 0.:
+                pywarpx.warpx.moving_window_dir = 'z'
+                pywarpx.warpx.moving_window_v = self.moving_window_velocity[2]/constants.c  # in units of the speed of light
```

Comment:
Is this OK for 2D and 3D?

Reply:
This bit of code is only for 2D. Thinking about it though, the index of moving_window_velocity should be 1 as it was before. I'll make undo this change.